server_installation/topics/operating-mode/crossdc/proc-configuring-remote-cache.adoc

```diff
@@ -0,0 +1,163 @@
+        <property name="infinispan.client.hotrod.trust_store_password">password</property>
+    </remote-store>
+</replicated-cache>
+----
+<1> Names the cache in the {jdgserver_name} configuration.
+<2> Names the corresponding cache on the remote {jdgserver_name} cluster.
+<3> Specifies the `remote-cache` socket binding.
++
+The preceding cache configuration includes recommended settings for {jdgserver_name} caches.
+Hot Rod client configuration properties specify the {jdgserver_name} user credentials and SSL keystore and truststore details.
++
+Refer to the
+ifeval::[{project_community}==true]
+https://infinispan.org/docs/11.0.x/titles/xsite/xsite.html#configure_clients-xsite[{jdgserver_name} documentation]
+endif::[]
+ifeval::[{project_product}==true]
+https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html-single/data_grid_guide_to_cross-site_replication/index#configure_clients-xsite[{jdgserver_name} documentation]
+endif::[]
+for descriptions of each property.
+
+. Add distributed caches to the Infinispan subsystem for each of the following caches:
++
+* sessions
+* clientSessions
+* offlineSessions
+* offlineClientSessions
+* actionTokens
+* loginFailures
++
+For example, add a cache named `sessions` with the following configuration:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<distributed-cache name="sessions" <1>
+                   owners="1"> <2>
+    <remote-store cache="sessions" <3>
+                  remote-servers="remote-cache" <4>
+                  passivation="false"
+                  fetch-state="false"
+                  purge="false"
+                  preload="false"
+                  shared="true">
+        <property name="rawValues">true</property>
+        <property name="marshaller">org.keycloak.cluster.infinispan.KeycloakHotRodMarshallerFactory</property>
+        <property name="infinispan.client.hotrod.auth_username">myuser</property>
+        <property name="infinispan.client.hotrod.auth_password">qwer1234!</property>
+        <property name="infinispan.client.hotrod.auth_realm">default</property>
+        <property name="infinispan.client.hotrod.auth_server_name">infinispan</property>
+        <property name="infinispan.client.hotrod.sasl_mechanism">SCRAM-SHA-512</property>
+        <property name="infinispan.client.hotrod.trust_store_file_name">/path/to/truststore.jks</property>
+        <property name="infinispan.client.hotrod.trust_store_type">JKS</property>
+        <property name="infinispan.client.hotrod.trust_store_password">password</property>
+    </remote-store>
+</distributed-cache>
+----
+<1> Names the cache in the {jdgserver_name} configuration.
+<2> Configures one replica of each cache entry across the {jdgserver_name} cluster.
+<3> Names the corresponding cache on the remote {jdgserver_name} cluster.
+<4> Specifies the `remote-cache` socket binding.
++
+. Start each {project_name} server, for example:
```

Comment:
@tkyjovsk @oraNod said that you were asking that I add a step to prepare separate/individual installations of KC for node11, node12, node21, and node22
Can you give more detail?

Reply:
@andymunro Please see the steps 4-9 from section section "Setting up Cross DC with Infinispan 9.4.19" subsection "Setting up Keycloak servers". These steps should be also present in the section for "Infinispan 11".

Reply:
@mposolda I think the steps for startup and verification of KC instances (see above) are exactly the same regardless of Infinispan version. Can you please confirm? If so, we can just copy the same steps in both sections.

Reply:
@mposolda @tkyjovsk I have added in the steps 4 to 9 from the Infinispan 9.4.19 version.  If you believe this does not work, let me know and I can change it.

Reply:
The step  `12. Start each {project_name} server, for example: ...` can be removed now. It was replaced by the previous steps (7-11).

Except for the block `If you notice the following warning messages ...`. This text should be appended to step 7 (after starting NODE11), including the log snippet with the warnings.